core/codecArrayBuffer.js

```diff
@@ -88,7 +88,36 @@ sjcl.codec.arrayBuffer = {
     return out;
   },
 
+  padBuffer: function (buffer, padding, padding_count) {
+    var ol, out, i;
+    padding = padding==undefined  ? true : padding
+    padding_count = padding_count || 16
 
+    if (buffer.byteLength === 0) {
+      return new ArrayBuffer(0);
+    }
+
+    ol = buffer.byteLength;
+    if (padding && ol%padding_count !== 0){
+      ol += padding_count - (ol%padding_count);
+    }
+
+    var out = new Uint8Array(ol);
+      out.set(new Uint8Array(buffer), 0);
+      return out.buffer;
+  },
+
+  /** Ciphertext is stored in sjcl encoded bitArray - this function converts it back to buffer **/
+  toBuffer: function (bitArray) {
```

Comment:
I am still not sure why you need this. Shouldn't the fromBits function do almost the same?